app/controllers/api/v1/users/sessions_controller.rb

```diff
@@ -2,6 +2,10 @@ class Api::V1::Users::SessionsController < Api::V1::BaseController
   def create
     load_resource
     if @user
+
+      @user.regenerate_session_token!
+      @user.regenerate_refresh_token!
```

Comment:
don't do this twice?

Reply:
It is actually for two separate tokens. The plan is the `session_token` expires quickly and is repeatedly replaced for as long as the `refresh_token` is valid to maintain access.